backend/api/Areas/Property/Models/Property/PropertyModel.cs

```diff
@@ -112,6 +112,16 @@ public class PropertyModel : BaseModel
         /// </summary>
         public TypeModel<string> AreaUnit { get; set; }
 
+        /// <summary>
+        /// get/set - District type.
+        /// </summary>
+        public TypeModel<short> DistrictType { get; set; }
+
+        /// <summary>
+        /// get/set - Region type.
+        /// </summary>
+        public TypeModel<short> RegionType { get; set; }
+
```

Comment:
nit: would prefer if these are named District, Region to align better with the "concept" Property. Ideally in the future we would want to refactor this specific model into the concept one